staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -158,15 +158,24 @@ type ObjectMeta struct {
 	// +optional
 	UID types.UID `json:"uid,omitempty" protobuf:"bytes,5,opt,name=uid,casttype=k8s.io/kubernetes/pkg/types.UID"`
 
-	// An opaque value that represents the internal version of this object that can
+	// A value that represents the internal version of this object that can
 	// be used by clients to determine when objects have changed. May be used for optimistic
 	// concurrency, change detection, and the watch operation on a resource or set of resources.
-	// Clients must treat these values as opaque and passed unmodified back to the server.
-	// They may only be valid for a particular resource or set of resources.
+	//
+	// This value is a string containing a base-10 encoding of a 63 bit positive integer.
```

Comment:
The status of this PR is that we're waiting for the SIG to hear the case for making this arbitrary-length.